OsmAnd/res/values/strings.xml

```diff
@@ -125,7 +125,7 @@
     <string name="quick_action_contour_lines_show">Show contour lines</string>
     <string name="quick_action_contour_lines_hide">Hide contour lines</string>
     <string name="quick_action_show_hide_contour_lines">Show/hide contour lines</string>
-    <string name="quick_action_hillshade_descr">A button to show or hide hillshades on the map.</string>
+    <string name="quick_action_hillshade_descr">A button to show or hide map hillshading.</string>
```

Comment:
We call them hillshades everywhere.

Reply:
My problem there is I somehow think it should be "show or hide hillshade on the map", without the s. Hills only cast one shade. Hillshades is otherwise the plural of many hills doing it, so that is fine.
As it is a layer, "throw/s hillshade on the map" just sounds functionally like what it does to me.
However, as an explanation, saying the same word twice doesn't do anything, so I thought this to be the better compromise.

Reply:
'Shades' (as in shades of color, or Shades of Grey') should be perfectly fine, I see no issues.

Reply:
I think I know why I don't like it now. "Shades" has lower resolution, effectively explaining dithering, rather than the organic concept of something casting a shade.
Map *shades sounds like the map is casting the shade, rather than the shade being cast onto the map.
They also are not called hillshades everywhere. (I did fix the ones that weren't).

```
<string name="quick_action_hillshade_descr">A button to show or hide map hillshading.</string>
<string name="quick_action_hillshade_show">Show hillshade</string>
<string name="quick_action_hillshade_hide">Hide hillshade</string>
<string name="quick_action_show_hide_hillshade">Show/hide hillshade</string>
<string name="contour_lines_and_hillshade">Contour lines and hillshade</string>
<string name="contour_lines_hillshade_maps">Contour lines and hillshade maps</string>
<string name="hillshade_menu_download_descr">Download the \'Hillshade Overlay\' map to show vertical shading.</string>
<string name="hillshade_purchase_header">Install the \'Contour lines\' plugin to show graded vertical areas.</string>
<string name="hillshade_layer_disabled">Hillshade layer disabled</string>
<string name="osmand_rastermaps_plugin_description">Access many types of online (so called tile or raster) maps, from predefined OSM tiles (like Mapnik) to satellite images, and special purpose layers like weather maps, climate maps, geological maps, hillshade layers, etc.\n\n
    Any of these maps can either be used as the main (base) map to be displayed, or as an overlay or underlay to another base map (like OsmAnd\'s standard offline maps). Certain elements of the OsmAnd vector maps can be hidden via the \'Configure map\' menu to make any underlay map more visible.\n\n
    Download tile maps directly online, or prepared them for offline use (manually copied to OsmAnd\'s data folder) as an SQLite database which can be produced by a variety of 3rd party map preparation tools.</string>
<string name="osmand_srtm_long_description_1000_chars">This plugin provides both a contour line overlay and a (relief) hillshade layer to be displayed on top of OsmAnd\'s standard maps. This functionality will be much appreciated by athletes, hikers, trekkers, and anybody interested in the relief structure of a landscape.\n\n
    The global data (between 70 ° north and 70 ° south) is based on measurements by SRTM (Shuttle Radar Topography Mission) and ASTER (Advanced Spaceborne Thermal Emission and Reflection Radiometer), an imaging instrument onboard Terra, the flagship satellite of NASA\'s Earth Observing System. ASTER is a cooperative effort between NASA, Japan\'s Ministry of Economy, Trade and Industry (METI), and Japan Space Systems (J-spacesystems).
</string>
<string name="srtm_plugin_description">This plugin provides both a contour line overlay and a (relief) hillshade layer to be displayed on top of OsmAnd\'s standard maps. This functionality will be much appreciated by athletes, hikers, trekkers, and anybody interested in the relief structure of a landscape. (Please note that contour line and/or relief data are separate, additional downloads available after activating the plugin.)
		\n\nThe global data (between 70 ° north and 70 ° south) is based on measurements by SRTM (Shuttle Radar Topography Mission) and ASTER (Advanced Spaceborne Thermal Emission and Reflection Radiometer), an imaging instrument onboard Terra, the flagship satellite of NASA\'s Earth Observing System. ASTER is a cooperative effort between NASA, Japan\'s Ministry of Economy, Trade and Industry (METI), and Japan Space Systems (J-spacesystems).
</string>
<string name="layer_hillshade">Hillshade layer</string>
<string name="download_hillshade_maps">Hillshade</string>
<string name="osmand_extended_description_part5">
        Cycling\n
        • Find cycle paths on the map\n
        • GPS navigation in cycling mode builds your route using cycle paths\n
        • See your speed and altitude\n
        • GPX recording option enables you to record your trip and share it\n
        • Via an additional plugin you can enable contour lines and hillshading
</string>
```